e2e-tests/tests/rearrange-elements.spec.ts

```diff
@@ -8,22 +8,31 @@ const expectedPage = fs.readFileSync(
 );
 
 studioTest("can rearrange elements in tree", async ({ page, studioPage }) => {
-  const banner = page.getByText("Banner");
-  const div = page.getByRole("list").filter({ hasText: "div" });
-  const divBox = await div.boundingBox();
+  await studioPage.switchPage("LocationPage");
+  await studioPage.addElement("Footer", "Components", false);
```

Comment:
Do we need to add a footer?

Reply:
Oh ok hmm I think I prefer doing that instead? Then we can skip the first screenshot

Reply:
I've been thinking we should avoid large full page screenshots where possible because they can result in false positives and we've also run into a number of "re-screenshot the world" scenarios when somebody adds a new UI element to the page